src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -15,92 +15,59 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Dict
+
 from d3a.d3a_core.singletons import bid_offer_matcher
+from d3a.models.area import Area
 from d3a.models.strategy.load_hours import LoadHoursStrategy
-from d3a.models.strategy.storage import StorageStrategy
 from d3a.models.strategy.pv import PVStrategy
+from d3a.models.strategy.storage import StorageStrategy
 from d3a_interface.constants_limits import ConstSettings
 
 
-class FileExportEndpoints:
-    def __init__(self):
-        self.plot_stats = {}
-        self.plot_balancing_stats = {}
-        self.cumulative_offers = {}
-        self.cumulative_bids = {}
-        self.clearing = {}
-
-    def __call__(self, area):
-        self._populate_area_children_data(area)
-
-    def _populate_area_children_data(self, area):
-        if area.children:
-            for child in area.children:
-                self._populate_area_children_data(child)
-        self.update_plot_stats(area)
+class PastDataEnum(Enum):
```

Comment:
I think the name of this class could be improved a bit: "past data" doesn't say much about the meaning of this class. What the class contains are actually the types of available markets that we have in the entire exchange.
I would vote for naming this class something like `AVAILABLE_MARKET_TYPES` (if it's not already used) and maybe using it in other parts of the code in the future (not just here in the export modules).